spec/components/guardian_spec.rb

```diff
@@ -2640,6 +2640,32 @@
     end
   end
 
+  describe '#can_ignore_user?' do
+    let(:guardian) { Guardian.new(user) }
+
+    context "when ignored user is the same as guardian user" do
+      it 'does not allow ignoring user' do
+        expect(guardian.can_ignore_user?(user.id)).to be_falsey
```

Comment:
We prefer `.to eq(false)` over `.to be_falsey`

Reply:
Is there a reason for the preference? – Also `be_falsey` is used everywhere in this spec, should we change it?

Reply:
`falsey` is not explicit enough as it'll match both `nil` and `false`.